v_4_0_0/filemanager.go

```diff
@@ -0,0 +1,54 @@
+package v_4_0_0
```

Comment:
I wonder if this code is really cloud config version dependent, maybe this should go one folder up so we don't copy the same code several times

Reply:
Probably, we can move it upper into some pkg/tooling package. But as the discussion goes on, we probably will make a new repo for this, so I would proceed with this. And when there will be a decision on structuring - we can change it. 
For now implementation can be unbounded from structure imo